packaging/os/pkgng.py

```diff
@@ -82,6 +82,15 @@
         required: false
         choices: [ "yes", "no" ]
         default: no
+    clean:
+        version_added: "2.2"
+        description:
+            - Clean the local cache of fetched remote packages. Use
+              C(yes) to only remove obsolete package files (e.g. old
+              versions) or C(all) to remove all package files.
+        required: false
+        choices: [ "yes", "no", "all" ]
```

Comment:
to not confuse people into thinking this is a boolean I would change the choices:

`'none', 'obsolete', 'all'`